src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -160,6 +161,18 @@ public class Session {
             HostToClientWorker(_process.StandardOutput.BaseStream, hostEnd).DoNotWait();
         }
 
+        private void StartSession() {
+            _process.Start();
+            _process.WaitForExit(250);
+            if (_process.HasExited && _process.ExitCode != 0) {
+                var message = ErrorCodeConverter.MessageFromErrorCode(_process.ExitCode);
```

Comment:
There are two kinds or errors: NTSTATUS which may be positive and Win32 (GetLastError) that are positive. Not sure if we ever get Win32 one as exit code. From NTSTATUS, those that we are interested in are indeed negative. I built a list of about 20 error code that probably matter.